vin/db/migrate/20150419210231_create_bottles.rb

```diff
@@ -1,10 +0,0 @@
-class CreateBottles < ActiveRecord::Migration
-  def change
-    create_table :bottles do |t|
-      t.integer :wine_id
-      t.integer :package_id
```

Comment:
Thanks man, i was sure there was a way of doing this nicely, didn't have the time to look at the documentation for it